src/main/java/seedu/address/logic/commands/EditTaskCommand.java

```diff
@@ -31,7 +36,8 @@ public class EditTaskCommand extends Command {
             + "Existing values will be overwritten by the input values.\n"
             + "Parameters: INDEX (must be a positive integer) "
             + "[" + PREFIX_TASK_TITLE + "TITLE] "
-            + "[" + PREFIX_TASK_NOTE + "NOTE]\n"
+            + "[" + PREFIX_TASK_NOTE + "NOTE]"
```

Comment:
Missing whitespace here also

```suggestion
            + "[" + PREFIX_TASK_NOTE + "NOTE] "
```